src/pages/resources.tsx

```diff
@@ -0,0 +1,66 @@
+import ResourceCard from "@components/page_components/resources/ResourceCard";
+import ResourceDrawer from "@components/page_components/resources/ResourceDrawer";
+import { TextInput } from "@mantine/core";
+import { useDebouncedValue, useDisclosure } from "@mantine/hooks";
+import { Resource } from "@ourtypes/resources_page_types/ResourceTypes";
+import { useEffect, useState } from "react";
+
+export default function ResourcesPage() {
+	const [resourceList, setResourceList] = useState<Resource[]>([]);
+	const [searchQuery, setSearchQuery] = useState("");
+
+	useEffect(() => {
+		fetch("https://center-be.stamford.dev/api/resources/resources").then((res) => {
```

Comment:
https://github.com/stamford-syntax-club/stamfordcenter-frontend/issues/35
XDDDD

Reply:
btw why don't you use SWR just like in `study_plans.tsx`?

Reply:
‘Cuz it doesn’t need to be updated constantly, swr is Stale While Revalidate, we don’t need to keep pulling in new data kekw

Reply:
not resolved!

DYNAMIC HOSTTTT

I want to see this in https://center-beta.stamford.dev
```
"name": "Class Schedules & Full Name Lists Beta Env งับ คริๆ",
```